hack/jenkins/job-configs/test-linkchecker.yaml

```diff
@@ -6,7 +6,9 @@
     logrotate:
         numToKeep: 200
     builders:
-        - shell: './hack/verify-linkcheck.sh'
+        - shell: |
```

Comment:
@ixdy PTAL :)

Not sure what does the "|" sign do here, I copied it from other configs and hopefully it's correct.